packages/nuqs/src/parsers.test.ts

```diff
@@ -48,6 +49,16 @@ describe('parsers', () => {
       ref
     )
   })
+  test('parseAsIsoDate', () => {
+    expect(parseAsIsoDate.parse('')).toBeNull()
+    expect(parseAsIsoDate.parse('not-a-date')).toBeNull()
+    const moment = '2020-01-01'
+    const ref = new Date(moment)
+    expect(parseAsIsoDate.parse(moment)).toStrictEqual(ref)
+    expect(parseAsIsoDate.parse(moment.slice(0, 10))).toStrictEqual(ref)
+    expect(parseAsIsoDate.serialize(ref).length).toBe(10)
+    expect(parseAsIsoDate.serialize(ref).length).not.toContain('T')
```

Comment:
suggestion: We could check for equality with the expected format here.

```suggestion
    expect(parseAsIsoDate.serialize(ref)).toEqual(moment)
```

Reply:
nice, yeah this cleans up and is more comprehensive a test. Thanks